src/commands/Social/pay.ts

```diff
@@ -64,14 +64,16 @@ export default class extends SkyraCommand {
 
 			return this.acceptPayment(message, language, user, money);
 		});
+
+		return message.sendMessage(response);
 	}
 
-	private async acceptPayment(message: GuildMessage, language: Language, user: User, money: number) {
+	private acceptPayment(message: GuildMessage, language: Language, user: User, money: number) {
 		this.client.emit(Events.MoneyPayment, message, message.author, user, money);
-		return message.sendMessage(language.get(LanguageKeys.Commands.Social.PayPromptAccept, { user: user.username, amount: money }));
+		return language.get(LanguageKeys.Commands.Social.PayPromptAccept, { user: user.username, amount: money });
```

Comment:
This won't return to the user?

Reply:
![image](https://user-images.githubusercontent.com/24852502/99068393-bcf97600-25ac-11eb-9a77-7e42e488c671.png)